.circleci/config.yml

```diff
@@ -29,6 +29,14 @@ jobs:
     steps:
     - checkout
 
+    # -mod=readonly is enabled by default starting in go1.16 causing many go
+    # tools and editor integrations to report problems if the go.sum file is
+    # not up-to-date, so let's make sure it is.
+    - run:
+        name: go.sum up-to-date
+        command: |
+          go run -mod=readonly gosum.go
```

Comment:
If you got better ideas for a way to check if `go.sum` is up-to-date, I'm all ears. I played around a bit, and this was the best I found.

Reply:
Can't we just run one of the other two steps using `go -mod=readonly`? Such as `checkmilestone.go` or `checkcopyright.go`, transferring the comment you left about this argument.

Reply:
Yes, I considered that. But I figured it's pretty misleading to have the `checkmilestone` step fail in the CI when it's really `go.sum` that's the problem. That being said, I don't feel too strongly, so if you prefer it I'll make the change.

Reply:
I agree. All good.